src/sass/_pattern_navigation.scss

```diff
@@ -144,6 +149,32 @@
   }
 }
 
+.accordian-nav-menu {
```

Comment:
```suggestion
.accordion-nav-menu {
```
typo, and there is another occurrence in `accordian-nav-secondary`.

Reply:
all accordian has been eliminated.